www/components/scroll/scroll.js

```diff
@@ -19,22 +19,29 @@ class scroll extends LitElement {
     // on hash change child event, scroll content
     const contains = (selector, text) => {
       const elements = this.querySelectorAll(selector);
+      // convert NodeList to Array
+      const elementsArray = Array.from(elements);
 
-      return Array.prototype.filter.call(elements, (element) =>{
+      return Array.prototype.filter.call(elementsArray, (element) => {
         return RegExp(text, 'gmi').test(element.textContent);
       });
     };
 
     let { hash } = window.location;
 
     // clean hash string, remove # and replace - with spaces
-    hash = hash.replace('#', '');
-    hash = hash.replace('-', ' ').toLowerCase();
-
+    hash = hash.replace('#', '').toLowerCase();
+    // for cases of multiple hyphens
+    while (hash.indexOf('-') !== -1) {
```

Comment:
str.replace(a, b) will replace all instances of a with b. There is no need to loop it.